src/card/card-header.jsx

```diff
@@ -94,14 +94,15 @@ const CardHeader = React.createClass({
     let titleStyle = this.prepareStyles(styles.title, this.props.titleStyle);
     let subtitleStyle = this.prepareStyles(styles.subtitle, this.props.subtitleStyle);
 
-    let avatar = this.props.avatar;
+    let avatar = this.props.avatar || null;
```

Comment:
Sorry, I made a mistake. I didn't see the part of the change below. Should I delete all my comment?